app/views/users/show.html.erb

```diff
@@ -44,7 +44,7 @@
 
         </h1>
         <p class="profile-description" itemprop="description">
-          <%= @user.summary.presence || ["404 bio not found"].sample %>
+          <%= @user.summary_html.present? ? @user.summary_html.html_safe : ["404 bio not found"].sample %>
```

Comment:
I think `sample` from an array of one value can be removed, it's probably a left over, thank you!